File: barcode-creator-core-console-cs/TwoDimensionalBarcode.cs
using ceTe.DynamicBarcode.Creator;
using System;
using System.IO;

namespace barcode_creator_core_console_cs
{
    class TwoDimensionalBarcode
    {
        public static void AztecBarcode(string outputPath)
        {
            Aztec barcode = new Aztec("Hello World", AztecSymbolSize.Full);
            byte[] barcodeData = barcode.Draw(96, 96, ImageFormat.Png);
            File.WriteAllBytes(outputPath, barcodeData);
        }
        public static void DataMatrix(string outputPath)
        {
            DataMatrixBarcode barcode = new DataMatrixBarcode("Hello World");
            barcode.Draw(outputPath, 300, ImageFormat.Png);
        }
        public static void MacroPdf417Barcode(string outputPath)
        {
            MacroPdf417 barcode = new MacroPdf417("Hello World", 3);
            barcode.Draw(outputPath, 300, ImageFormat.Png);
        }

        public static void Pdf417Barcode(string outputPath)
        {
            Pdf417 barcode = new Pdf417("Hello World", 3);
            barcode.Draw(outputPath, 300, ImageFormat.Png);
        }

        public static void QRCodeBarcode(string outputPath)
        {
            QRCode barcode = new QRCode("Hello World");
            barcode.Draw(outputPath, 300, ImageFormat.Png);
        }
    }
}

File: barcode-creator-framework-console-vb/TwoDimensionalBarcode.vb
Imports System.IO
Imports ceTe.DynamicBarcode.Creator

Public Class TwoDimensionalBarcode
    Public Shared Sub AztecBarcode(ByVal outputPath As String)
        Dim MyBarCode As Aztec = New Aztec("Hello World", AztecSymbolSize.Full)
        Dim MyBarCodeData As Byte() = MyBarCode.Draw(96, 96, ImageFormat.Png)
        File.WriteAllBytes(outputPath, MyBarCodeData)
    End Sub

    Public Shared Sub DataMatrix(ByVal outputPath As String)
        Dim MyBarCode As DataMatrixBarcode = New DataMatrixBarcode("Hello World")
        MyBarCode.Draw(outputPath, 300, ImageFormat.Png)
    End Sub

    Public Shared Sub MacroPdf417Barcode(ByVal outputPath As String)
        Dim MyBarCode As MacroPdf417 = New MacroPdf417("Hello World", 3)
        MyBarCode.Draw(outputPath, 300, ImageFormat.Png)
    End Sub

    Public Shared Sub Pdf417Barcode(ByVal outputPath As String)
        Dim MyBarCode As Pdf417 = New Pdf417("Hello World", 3)
        MyBarCode.Draw(outputPath, 300, ImageFormat.Png)
    End Sub

    Public Shared Sub QRCodeBarcode(ByVal outputPath As String)
        Dim MyBarCode As QRCode = New QRCode("Hello World")
        MyBarCode.Draw(outputPath, 300, ImageFormat.Png)
    End Sub
End Class
